plugins/router/f5/f5.go

```diff
@@ -712,6 +825,11 @@ func (f5 *f5LTM) Initialize() error {
 		}
 	}
 
+	err = f5.ensurePartitionPathExists(f5.partitionPath)
```

Comment:
I moved the check up to the top. 
Hmm, re: specifying f5.partitionPath in the other places, will need to check on the docs if there is a way to do it. The docs seemed quite sketchy on that.